api/slack.go

```diff
@@ -6,24 +6,22 @@ import (
 	"github.com/slack-go/slack"
 )
 
-func SendSlackMessages(slackToken string, messages map[string]string) {
-	log := logger.Get()
+type SlackClientInterface interface {
+	PostMessage(channelID string, options ...slack.MsgOption) (string, string, error)
+}
 
-	if len(slackToken) > 0 {
```

Comment:
I'm just noticing, we seem to have lost all checks for a Slack token. So now we always try to send to Slack even if we know that we can't.

Reply:
Yeah, good catch, let me fix that!

Reply:
Fixed and a unit test was provided.